core/examples/json/crawlable_catalog/dataset_example/nz-buildings-collection.json

```diff
@@ -51,9 +51,17 @@
       ]
     },
     "formats": [
-      "GeoPackage",
-      "GeoJSONSeq",
-      "FlatGeobuf"
+      {
+        "name": "GeoPackage",
+        "mediaType": "application/geopackage+sqlite3"
+      },
+      {
+        "name": "GeoJSONSeq",
+        "mediaType": "application/geo+json"
```

Comment:
Should this be `application/geo+json-seq` as per https://datatracker.ietf.org/doc/html/rfc8142#section-5?

Reply:
Fixed.